streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/message/CallResult.java

```diff
@@ -0,0 +1,122 @@
+package io.ray.streaming.runtime.message;
+
+import com.google.common.base.MoreObjects;
+import java.io.Serializable;
+
+public class CallResult<T> implements Serializable {
```

Comment:
Why we need this class? Need more documents

Reply:
I'll try to remove this class.

Reply:
I'll do this in separate PR.